### PR TITLE
PP-277: Multinode jobs may fail to start

### PR DESCRIPTION
**Pull request by mike0042**
#### Issue
* PP-277

#### Problem
* A sister mom fails to find itself in the node list on multihomed systems.

#### Cause
* The node list created by the server referenced an alternate node name.

#### Solution
* Added PBS_MOM_NODE_NAME as a way of forcing mom to use the name we want.

### CODE REVIEW

File: pbspro.spec [hunk 1]
```diff
@@ -194,7 +196,9 @@ HPC clusters, clouds and supercomputers.
 This package is intended for a client host and provides
 the PBS Professional user commands.
 
-%if 0%{!?opensuse_bs}
+%if 0%{?opensuse_bs}
```
Comment (subhasisb) [inline pbspro.spec]: Is this change related to the ticket being addressed in this pull request? If not, I suggest we create another ticket to address this.

File: pbspro.spec [hunk 2]
```diff
@@ -50,9 +50,11 @@ Name: %{pbs_name}
 Version: %{pbs_version}
 Release: %{pbs_release}
 Source0: %{pbs_dist}
+%if 0%{?opensuse_bs}
```
Comment (subhasisb) [inline pbspro.spec]: Is this change related to the ticket being addressed in this pull request? If not, I suggest we create another ticket to address this.

File: pbspro.spec (hunk 1 above)
Comment (mike0042) [inline pbspro.spec]: The problem was identified while building and testing this ticket, but it's not strictly related. Seemed like a lot of overhead to file a separate ticket, so I just made the change here.

File: pbspro.spec (hunk 2 above)
Comment (mike0042) [inline pbspro.spec]: The problem was identified while building and testing this ticket, but it's not strictly related. Seemed like a lot of overhead to file a separate ticket, so I just made the change here.

**Comment by subhasisb** [comment]
Looks good I sign off

**Pull request merged**